types/react-native-form/index.d.ts

```diff
@@ -9,7 +9,6 @@ export interface FormProps extends ViewProps {
             valueProp: string;
         };
     } | undefined;
-    ref: string;
```

Comment:
`ref` is a reserved prop by the version React that react-native uses. It is not up to the component to override this type. The `ref` type now solely comes from `ViewProps`